tests/es6/activities/func.js

```diff
@@ -12,6 +12,9 @@ let _ = require("lodash");
 let errors = wf4node.common.errors;
 
 describe("Func", function () {
+    before(function () {
+        activityMarkup.registerTypes('../../../lib/es6/activities/index');
```

Comment:
I think this will break the ability for automation-core to override w4b activities.
https://github.com/CrossChx/workflow-4-node/commit/5bcbb49ed27757bae395093fe149e61b2beffaa1

Reply:
Not sure I agree? Since this is just in the test code, it should not be required in core.

Reply:
Ok.